### DISCUSSION TITLE
Use distinct iconography for PRs with merge conflicts

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#14307] [Pull Requests]
Here's a view of some PRs:
<img width="42" alt="image" src="https://user-images.githubusercontent.com/2119212/161675477-cb3b45eb-51b9-4019-ba13-d375d1713b12.png">

The third one is clearly a draft.

You can't tell, but the last 2 have merge conflicts, which if you visit them, appear as:
<img width="48" alt="image" src="https://user-images.githubusercontent.com/2119212/161675530-df9289f3-a1a7-4352-9a01-c506fb806e0b.png">
<img width="760" alt="image" src="https://user-images.githubusercontent.com/2119212/161675553-3d68e014-3f9e-484a-8fea-d53fe52c642a.png">

Personally, I'd suggest an icon w/ an✖️ in the top right.

For a short list of 6, it isn't a big deal to click through to discover that some PRs are stuck, but for larger lists, it's considerably more expensive.

Note: there is no `is:mergeable` option in [filters](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests) -- it'd definitely be nice if there was a way to filter out such items (or filter for them...)

**Comment by Zopolis4** [comment]
There's also no distinction when hovering over the icons-- they just say `Open pull request` or `Open draft pull request`, so something like `Open (draft) pull request with merge conflicts` for conflicting PR's would be nice.

**Comment by SCG82** [comment]
Change the color to red if there's a conflict.

**Reply by Zopolis4** [reply]
Those conflict with closed PR's though.
![image](https://user-images.githubusercontent.com/64618338/172731012-8de5f86c-940e-4c10-8020-cbf796dd40e7.png)

**Reply by SCG82** [reply]
how about `var(--color-attention-fg)` (`#9a6700` in light mode), or `var(--color-attention-emphasis)` (`#bf8700` in light mode)
